websites/S/Streamlabs/dist/metadata.json

```diff
@@ -12,12 +12,10 @@
 	"url": [
 		"streamlabs.com",
 		"dev.streamlabs.com",
-		"howto.streamlabs.com",
-		"ideas.streamlabs.com",
 		"support.streamlabs.com"
 	],
-	"version": "1.1.13",
-	"logo": "https://i.imgur.com/LvPRxYG.png",
+	"version": "1.1.14",
```

Comment:
according to the PR description and the code this seems like a breaking change to me is it not?